pom.xml

```diff
@@ -597,6 +597,7 @@ limitations under the License.
                 <groupId>com.codahale.metrics</groupId>
                 <artifactId>metrics-graphite</artifactId>
                 <version>${metrics.version}</version>
+                <optional>true</optional>
```

Comment:
Doh, I probably only focused on the addition of the optional tag and assumed it's the same dependency block as above. All good here.